app/components/navs/nav-rail.jsx

```
@@ -219,25 +218,48 @@
-                    localStorage.setItem('nav-rail-extended', JSON.stringify(!extended));
-                    setExtended(!extended);
-                }}>
-                    <div className={`state-layer rounded-full flex gap-4 relative padding-${extended ? "extended" : "collapsed"}`}>
-                        <md-ripple/>
-                        <span className="material-symbols-outlined block w-6">menu</span>
-                        <span className={`extended-${extended}`}>Menu</span>
+            <div className={`h-full ${extended ? 'p-4' : 'p-3'}`}>
+                <button 
+                    className="w-full rounded-full text-[--md-sys-color-on-surface] hover:bg-[--md-sys-color-surface-container]"
+                    role="button" 
+                    onClick={() => { 
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)